### PR TITLE
Add author class unit test

### PR DESCRIPTION
**Pull request by Eccb7**
In this pull request, I have added the tests on the author class which is passing.

### CODE REVIEW

File: classes/author.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+class Author
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
+  end
+
+  def add_item(item)
+    item.set_author(self)
+  end
```
Comment (henokkhm) [inline classes/author.rb:12]: Please pull the changes from the other branch. I have already done a code review on this.

File: classes/game.rb [hunk 2]
```diff
@@ -1,7 +1,8 @@
-class Game
+class Game < Item
   attr_accessor :title, :multiplayer, :last_played_at
 
   def initialize(title, multiplayer, last_played_at)
+    super()
```
Comment (henokkhm) [inline classes/game.rb:5]: The constructor of the Item class expects two arguments.
Currently, trying to create a Game object throws the following error.

image

File: spec/author_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  it 'has a name attribute' do
+    expect(author.name).to eq('Stephen King')
+  end
+
+  it "can add an item to the author's collection" do
+    # Create a sample item
+    item = double('Item')
+
+    expect(item).to receive(:set_author).with(author)
+
```
Comment (henokkhm) [inline spec/author_spec.rb:16]: I think you can add a test that checks that the **item** is added to the **items** array when the add_item method is called.

**Review by henokkhm — CHANGES_REQUESTED** [review]
Hi @Eccb7  Good job thus far. Please address the issues below and we can move on with implementing the remaining features.  Currently, one cannot create a Game object because an error is thrown. Thanks!

**Comment by Eccb7** [comment]
Hi @henokkhm 
You can check this branch and re-review it please

**Review by henokkhm — APPROVED** [review]
Hi @Eccb7 

## Status: Approved

Thanks for making the changes! You can now merge your code.

**Pull request merged**